client/coral-embed-stream/src/containers/Embed.js

```diff
@@ -8,31 +8,41 @@ import renderComponent from 'recompose/renderComponent';
 
 import {Spinner} from 'coral-ui';
 import {authActions, assetActions, pym} from 'coral-framework';
-import {getDefinitionName, separateDataAndRoot} from 'coral-framework/utils';
+
+import Stream from './Stream';
 import Embed from '../components/Embed';
-import {setCommentCountCache, viewAllComments} from '../actions/stream';
+
 import {setActiveTab} from '../actions/embed';
-import Stream from './Stream';
+import {addExternalConfig} from 'coral-framework/actions/config';
+import {setCommentCountCache, viewAllComments} from '../actions/stream';
+import {getDefinitionName, separateDataAndRoot} from 'coral-framework/utils';
 
 const {logout, checkLogin} = authActions;
 const {fetchAssetSuccess} = assetActions;
 
 class EmbedContainer extends React.Component {
-
   componentDidMount() {
     pym.sendMessage('childReady');
+
+    pym.onMessage('config', config => {
+      this.props.addExternalConfig(JSON.parse(config));
+    });
```

Comment:
This is too late. The config must be available before any Components are rendered. Also how do you plan on applying the configs like `asset_id` to our components?